src/pages/Address/AddressAssetComp.tsx

```diff
@@ -119,7 +141,15 @@ export const AddressMNFTComp = ({ account }: { account: MNFT }) => {
   )
 }
 
-export const AddressNRC721Comp = ({ account }: { account: NRC721 }) => {
+export const AddressNRC721Comp = ({
+  account,
+  isRGBPP,
+  udtLabel,
```

Comment:
It's `nrc 721 comp` so the label should be `nrc 721`

Reply:
done